### PR TITLE
Environment: Add network

### PR DESCRIPTION
**Pull request by PatricioHenderson**

### CODE REVIEW

File: src/Networks.ts [hunk 1]
```diff
@@ -47,3 +47,13 @@ export function isFantom(): boolean {
 export function isGnosis(): boolean {
   return dataSource.network() == 'gnosis'
 }
+
+export function getNativeNetwork(): string {
+  if (isEthNetwork()) return 'Ethereum'
+  if (isMaticNetwork()) return 'Matic'
```
Comment (facuspagnuolo) [inline src/Networks.ts:53]: Careful, these two will return true on different networks, we should handle one by one.

File: src/Networks.ts [hunk 2]
```diff
@@ -47,3 +47,17 @@ export function isFantom(): boolean {
 export function isGnosis(): boolean {
   return dataSource.network() == 'gnosis'
 }
+
+export function getNativeNetwork(): string {
```
Comment (facuspagnuolo) [inline src/Networks.ts:51]: Can we rename it to `getNetworkName`?

**Review by facuspagnuolo — APPROVED** [review]
LGTM!

File: src/Networks.ts (hunk 2 above)
Comment (PatricioHenderson) [reply src/Networks.ts]: Sure! 
Done.

**Pull request merged**